libs/payments/ui/src/lib/client/components/LoadingSpinner/en.ftl

```diff
@@ -0,0 +1,2 @@
+payments-client-loading-spinner.aria-label = Loading…
+payments-client-loading-spinner.alt = Loading…
```

Comment:
```suggestion
payments-client-loading-spinner =
    .alt = Loading…
```

Reply:
these both can be combined https://projectfluent.org/fluent/guide/attributes.html

Reply:
oh yes, thanks for pointing that out (I didn't realize those were the same IDs, yes they should be combined).